### DISCUSSION TITLE
¿Cuál es la diferencia entre "Crea tu primer Wireframe UX" y "Ahora el UI" ?

### DISCUSSION THREAD

**Discussion by 358b06c6** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#156] [Q&A]
Veo que en ambas prácticas se pueden utilizar las mismas herramientas, tengo entendido que el Wireframe UX es como el boceto de como se usaría la aplicación; lo que me causa conflicto es la práctica de _Ahora el UI_ según yo son la misma cosa.

**Comment by AngelCruzO** [comment]
El UX es como es ver cómo acomodar tus componentes y que componentes utilizarías. Mientras que el UI ya es como funcionaria, se colocan colores, iconos, tipografía, interacciones, etc.

**Comment by DaveHdz13** [comment]
El UX solo tiene que explicar cómo va a funcionar tu página (botones, directorios, menús, etc.)

![Ux](https://user-images.githubusercontent.com/76536169/155899222-861b98f8-cfa1-42cb-b7f5-92d55933a32b.png)

El UI es la parte bonita y en donde vas a decidir: tamaños, colores, imágenes, etc. 

![UI](https://user-images.githubusercontent.com/76536169/155899269-d8b79271-70dc-4cae-8cec-d509647a7ad9.png)

**Reply by OscarIslasH** [reply]
Se pueden hacer ambas juntas? Hice el UI y ahi estoy explicando para que sirve cada boton desde ahi

https://miro.com/app/board/uXjVOJwm_Ic=/?invite_link_id=161925078655

**Reply by JuanSalvador73** [reply]
Gracias por el tip.

**Reply by OscarIslasH** [reply]
@JuanSalvador73 Fue mas una pregunta jajaja, pero si te sirve como ejemplo no hay de que, si tienes dudas de como voy haciendolo o de alguna cosa te puedo apoyar desde aqui, ya casi acabo

**Reply by DaveHdz13** [reply]
Si @OscarIslasH, de hecho en Figma puedes hacer las dos 👍  o también en Miro como lo estas haciendo, la verdad te esta quedando bien eh con todo y vista móvil esta muy bien 😄 👍.

**Reply by OscarIslasH** [reply]
@DaveHdz13 

Gracias! Ahi la voy mejorando de poco en poco, le fui agarrando el gusto

**Reply by JossPerezT** [reply]
Gracias @DaveHdz13. 
Hice un UX pero me está gustando más como queda mi UI con interacciones y ya planeaba hacer un nuevo UX con las modificaciones 🥴😂
